### ISSUE TITLE
LocalFree should be take `void*` or IntPtr as its parameter

### ISSUE THREAD

**Issue by AArnott** [microsoft/CsWin32#709]
## Actual behavior

The `LocalFree` method is declared by CsWin32 as taking an `nint`:

```cs
[DllImport("KERNEL32.dll", ExactSpelling = true, SetLastError = true)]
[DefaultDllImportSearchPaths(DllImportSearchPath.System32)]
internal static extern nint LocalFree(nint hMem);
```

## Expected behavior

The method takes a *pointer*. Not an arbitrary pointer-sized integer. While this is just semantics, it is more intuitive to use `IntPtr` or (even better) `void*` for the parameter type.
As it is, I have to cast pointers to `nint` to call this method.

Same for the return type.

**Comment by AArnott** [comment]
Blocked by https://github.com/microsoft/win32metadata/issues/1296

**Comment by AArnott** [comment]
Fixed in #889
